Validation/MtdValidation/plugins/BuildFile.xml

```diff
@@ -4,4 +4,5 @@
 <use name="Geometry/Records"/>
 <use name="Geometry/MTDGeometryBuilder"/>
 <use name="DQMServices/Core"/>
+<use name="hepmc"/>
```

Comment:
@casarsa why this dependence?

Reply:
That surprised me too. It seems it's required by SimDataFormats/CrossingFrame/interface/CrossingFrame.h, that includes HepMCProduct.h.

Reply:
By quick look it looks like the include of `HepMCProduct.h` could easily be removed from `SimDataFormats/CrossingFrame/interface/CrossingFrame.h`.

Reply:
Regardless of the possible cleanup of `SimDataFormats/CrossingFrame/interface/CrossingFrame.h`, the proper dependence to be added here is `SimDataFormats/CrossingFrame`. That should make the explicit dependence in `hepmc` uneecessary (as `SimDataFormats/CrossingFrame` depends on that).

Reply:
@makortel let's open a separate issue for that